lib/go/test/go.mod

```
@@ -4,91 +4,111 @@ go 1.18
 
 require (
 	github.com/coreos/go-semver v0.3.0
-	github.com/onflow/cadence v0.39.12
-	github.com/onflow/flow-core-contracts/lib/go/contracts v1.2.4-0.20230703193002-53362441b57d
-	github.com/onflow/flow-core-contracts/lib/go/templates v1.2.3
-	github.com/onflow/flow-emulator v0.51.2-0.20230704183611-ecad54e231b7
-	github.com/onflow/flow-go v0.31.1-0.20230704154018-87a84e9d36c2
-	github.com/onflow/flow-go-sdk v0.41.6
-	github.com/onflow/flow-go/crypto v0.24.7
+	github.com/onflow/cadence v0.42.5
+	github.com/onflow/flow-core-contracts/lib/go/contracts v1.2.4-0.20231016154253-a00dbf7c061f
+	github.com/onflow/flow-core-contracts/lib/go/templates v1.2.4-0.20231016154253-a00dbf7c061f
```

```suggestion
	// replaced by module version in this repo - disregard pinned version
	github.com/onflow/flow-core-contracts/lib/go/contracts v1.2.4-0.20231016154253-a00dbf7c061f
	// replaced by module version in this repo - disregard pinned version
	github.com/onflow/flow-core-contracts/lib/go/templates v1.2.4-0.20231016154253-a00dbf7c061f
```

We could put these two in a separate `require` block next to the `replace` block at the bottom to make this fact more obvious.

---

https://github.com/onflow/flow-core-contracts/pull/397/commits/13605a229b143d3d715cc2bd276f4a518df261e5

Btw, the imported version of `flow-core-contract` was updated automatically after updating the imported `flow-go` version (which itself imports `flow-core-contract`)